src/Interpreters/Context.cpp

```diff
@@ -3687,6 +3687,29 @@ ThrottlerPtr Context::getMergesThrottler() const
     return shared->merges_throttler;
 }
 
+void Context::reloadRemoteThrottlerConfig(size_t read_bandwidth, size_t write_bandwidth) const
```

Comment:
There will be 3 cases for the remote throttler:

1. The current logic: If the server start with bandwidth = 0, the throttler will not be initialized and throttler = nil.
2. The current logic: If the server start with bandwidth > 0, the throttler will be initialized.
3. The new feature: Changing the bandwidth on fly, including resetting it to 0.

My approach aims to preserve the case 1, with minimal impact on the existing default logic. Specifically, if the server start with bandwidth = 0, the throttler will remain nil.